packages/fiori/src/IllustratedMessage.ts

```diff
@@ -364,6 +379,16 @@ class IllustratedMessage extends UI5Element {
 		}
 	}
 
+	_setTitleAccAttrs() {
+		const title = this.shadowRoot!.querySelector(".ui5-illustrated-message-root ui5-title") || this.querySelector("ui5-illustrated-message ui5-title");
```

Comment:
It will be better to set level to use the getter that renders the slot element `hasFormattedTitle`. Since the slot has HTMLElement type we can't add `level` attribute to every element. Also is it correct to set aria-label attribute to ui5-title since it has level attribute?

Reply:
To make it easy the new property better only effect the default ui5-title rendered by the illustrated message, the slotted one could be changed by apps already. Can be directly passed in the illustrated message .hbs file.